File: doc/snippets/whatsapp/message/reaction/snippet.py
import requests
import json

url = 'https://api.smsbox.pro/waba/json/<WABAID>/reaction/<PHONEID>'
headers = {
    'Authorization': 'App <APIKEY>',
    'Content-Type': 'application/json'
}
data = {
    "recipients": [
        "+336XXXXXXXX"
    ],
    "contentMessage": {
        "type": "reaction",
        "reaction": {
            "message_id": 'wamid.xxxxxxxxxxxxxxxxxxxxxxxxxxxx',
            "emoji": "😀"
        }
    }
}

response = requests.post(url, headers=headers, data=json.dumps(data))

print(response.status_code)
print(response.json())

File: doc/snippets/whatsapp/message/reaction/snippet.php
<?php

$url = 'https://api.smsbox.pro/waba/json/<WABAID>/reaction/<PHONEID>';
$headers = [
    'Authorization: App <APIKEY>',
    'Content-Type: application/json'
];
$data = [
    'recipients' => [
        "+336XXXXXXXX"
    ],
    "contentMessage" => [
        "type" => "reaction",
        "reaction" => [
            "message_id" => 'wamid.xxxxxxxxxxxxxxxxxxxxxxxxxxxx',
            "emoji" => "😀"
        ]
    ]
];

$ch = curl_init($url);
curl_setopt($ch, CURLOPT_HTTPHEADER, $headers);
curl_setopt($ch, CURLOPT_POST, 1);
curl_setopt($ch, CURLOPT_POSTFIELDS, json_encode($data));
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);

$response = curl_exec($ch);
curl_close($ch);

echo $response;
